packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -241,16 +243,31 @@ const ZoomOutTooltip = () => (
   </>
 );
 
+const isLocaleFormatEnable = config.featureToggles.localeFormatPreference;
+const localeOptions: Intl.DateTimeFormatOptions = {
+  day: '2-digit',
+  month: '2-digit',
+  year: 'numeric',
+  hour: '2-digit',
+  minute: '2-digit',
+  second: '2-digit',
+};
```

Comment:
This seems to be used in a few places, could it be saved to a constant file or something like that?